### ISSUE TITLE
standard import "from six.moves.urllib.parse import quote" should be placed before "import bottle" (wrong-import-order)

### ISSUE THREAD

**Issue by mgedmin** [PyCQA/pylint#2181]
### Steps to reproduce
1. Create a virtualenv using Python 3
2. Install `pylint`, `six` and `bottle` into the virtualenv
3. Create a `wtf.py` with the following two lines

```python
import bottle
from six.moves.urllib.parse import quote
```

4. Run `pylint wtf.py`

### Current behavior

```
...
C:  2, 0: standard import "from six.moves.urllib.parse import quote" should be placed before "import bottle" (wrong-import-order)
```

### Expected behavior

pylint ought to realize `six` is a 3rd-party library and not a standard library module.

### pylint --version output

```
pylint 1.9.2, 
astroid 1.6.5
Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0]
```

(Note that this happens only on Python 3.)

**Comment by mgedmin** [comment]
Note that adding `six` to `known-third-party` in .pylintrc appears to be insufficient to shut this warning up.
